authui/src/authflowv2/base.css

```diff
@@ -33,6 +33,9 @@
     --color-primary-600: #052e6f;
     --color-primary-700: #011234;
 
+    /* secondary */
+    --color-text-secondary: #7e747b;
```

Comment:
It seems text/secondary is grey/base.

<img width="394" alt="Screenshot 2024-01-12 at 1 21 29 PM" src="https://github.com/authgear/authgear-server/assets/34942210/5bd100cc-e0a8-4bd0-8c8a-0425ae2eeaf2">

(Or actually we need three levels? grey/base -> text-secondary -> alert-message__text_color)